lang/java/reef-tests/src/test/java/org/apache/reef/tests/LocalTestEnvironment.java

```diff
@@ -73,7 +73,7 @@ public synchronized void tearDown() {
 
   @Override
   public int getTestTimeout() {
-    return 60000; // 1 min.
+    return 1200000; // 2 min.
```

Comment:
That's 20 minutes, actually. Remove one extra zero?

Reply:
Addressed